Index: js/CompileTree.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- js/CompileTree.js	(revision )
+++ js/CompileTree.js	(revision )
@@ -103,13 +103,15 @@
             var liObj = this;
             while (liObj) {
                 if (liObj._attr) {
-                    for (var k in fields) {
-                        var propName = fields[k];
+                    for (var attr in fields) {
+                        for (var k in fields[attr]) {
+                            var propName = fields[attr][k];
-                        if (props[propName]) {
+                            if (props[propName]) {
 
-                        } else {
-                            if (liObj._attr[propName]) {
-                                props[propName] = liObj._attr[propName];
+                            } else {
+                                if (liObj._attr[propName]) {
+                                    props[propName] = liObj._attr[propName];
+                                }
                             }
                         }
                     }
